Competition2018/src/test/java/competition/BaseTest.java

```diff
@@ -0,0 +1,26 @@
+package competition;
+
+import org.junit.Ignore;
+
+import xbot.common.injection.BaseWPITest;
+import xbot.common.injection.UnitTestModule;
+
+@Ignore
+public class BaseTest extends BaseWPITest {
```

Comment:
Let's call this `BaseCompetitionTest` - `BaseTest` sounds like it is the parent of `BaseWPITest`